client/src/components/RelatedProducts/Outfits/Outfit.jsx

```diff
@@ -0,0 +1,77 @@
+import { React, useState } from 'react';
+import styled from 'styled-components';
+import { useCurrentProduct } from '../../../contexts/ProductIDContext';
+
+import OutfitCard from './OutfitCard';
+
+// to check functionality of 'add to outfit'
+// will need to add current product to 'Your Outfit' list
+
+// change image to cover
+export default function Outfit() {
+  const productToAdd = useCurrentProduct().currentProduct;
+  const initialArray = [];
+  const [outfitArray, setOutfitArray] = useState(initialArray);
+
+  return (
+
+    <AddSection>
+      <AddButton>
+
+        <img
```

Comment:
Is it possible to refine the thumbnail display so its ratio is not distorted?

Reply:
I'll edit the thumbnail photo CSS, so it's not distorted when I start doing the final CSS. Walter suggested 'object-fit: cover'